.github/workflows/e2e_test.yml

```
@@ -0,0 +1,30 @@
+name: e2e_test
+on:
+  schedule:
+    # Run every night.
+    - cron:  '0 0 * * *'
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-18.04
```

Let's pin the version or use a Docker container.